sdk/iOS/src/MSPullSettings.m

```diff
@@ -0,0 +1,37 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ----------------------------------------------------------------------------
+
+#import "MSPullSettings.h"
+
+#pragma mark * MSPullSettings Implementation
+
+@implementation MSPullSettings
+
+static NSInteger const MSDefaultPageSize = 50;
+
+#pragma mark * Initializer method(s)
+
+- (instancetype)init {
```

Comment:
I think we'd only have constructors for the commonly used setups (page size being one of them)  I'd be fine with not having one that includes the QoS param when that is added.